EIPS/eip-7541.md

```diff
@@ -0,0 +1,139 @@
+---
+eip: 7541
+title: Apply Coinbase Rewards At The End Of Block
+description: Apply rewards for all transactions to the coinbase at the end of the block.
+author: Roman Krasiuk (@rkrasiuk), Dragan Rakita (@rakita)
+discussions-to: https://ethereum-magicians.org/t/eip-7541-apply-coinbase-rewards-at-the-end-of-block/16180
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-09-19
+---
+
+## Abstract
+
+The transaction fees shall be added to the `COINBASE` balance at the end of the block.
+
+## Motivation
+
+Currently, the transaction fees are applied to the `COINBASE`'s balance at the end of each transaction thus introducing an implicit dependency on each preceding transaction in the block. Moving the transaction reward to the end of the block removes this dependency allowing for various flavors of parallel execution.
+
+## Specification
+
+The sender of the transaction is still credited at the start of the transaction execution and the refund is still issued at the end in order to avoid double-spending. The `COINBASE` reward is accumulated from transaction fees and applied at the end of each block.
```

Comment:
```suggestion
The sender of the transaction is still charged at the start of the transaction execution and the refund is still issued at the end in order to avoid double-spending. The `COINBASE` reward is accumulated from transaction fees and applied at the end of each block.
```
I think you mean charged here?